Main.java

```diff
@@ -1,15 +1,25 @@
+import java.util.Scanner;
 /**
- *
- * @author 
+ * To convert the temperature from celsius to fahrenheit
+ * @ThomasQuigley 
  */
 public class Main {
 
-  /**
-   * The method that is executed when you hit the run button.
-   * @param args the command line arguments
-   */
+
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // creating the scanner
+    Scanner input = new Scanner(System.in);
+
+    // ask for the temperature they want to convert
+    System.out.println("Please enter a temperature you would like to convert in degrees Celsius.");
+    
+    int celsius = input.nextInt();
+
+    // convert the tempature to fahrenheit
+    int fahrenheit = (celsius * 9)/5 + 32;
```

Comment:
Great work. celsius and fahreneit are OK names but they are really the unit attached to the value. It would be better to use something like tempInC and tempInF. It works, but it could be a little better. Otherwise, excellent job!